internal/rbd/controllerserver.go

```diff
@@ -1447,8 +1447,7 @@ func (cs *ControllerServer) ControllerExpandVolume(
 		return nil, status.Error(codes.InvalidArgument, err.Error())
 	}
 	defer cr.DeleteCredentials()
-
-	rbdVol, err := GenVolFromVolID(ctx, volID, cr, req.GetSecrets())
+	rbdVol, err := genVolFromVolIDWithMigration(ctx, volID, cr, req.GetSecrets())
```

Comment:
genVolFromVolIDWithMigration return nil in case of err so `defer rbdVol.Destroy()` will panic